packages/stylelint-plugin/README.md

```diff
@@ -32,7 +31,13 @@ These rules are included by default:
 - `kaizen/no-invalid-use-of-var-tokens-in-functions` - Detects invalid usages of Kaizen variable tokens within some functions that are known to be problematic such as `rgba`, `darken` or `mix`, and automatically fixes `rgba`, `rgb`, `add-alpha`, `transparentize` instances by ensuring the correct `-rgb-params` tokens are used.
 - `kaizen/no-invalid-use-of-var-tokens-in-equations` - Detects invalid usages of Kaizen tokens within equations. It won't be able to automatically fix many cases, but it can for some simple cases for example negating a Kaizen variable without a runtime `calc()` function.
 - `kaizen/all-token-imports-must-be-used` - Reports and fixes unused Kaizen token imports.
-- `kaizen/tokens-must-be-imported` - Reports and fixes any Kaizen token imports that should exist but don't, based on the tokens that are used in the current stylesheet.
+- `kaizen/all-used-tokens-must-be-imported` - Reports and fixes any Kaizen token imports that should exist but don't, based on the tokens that are used in the current stylesheet.
+- `kaizen/rename-tokens` - Reports and automatically renames any ex-generation or deprecated color names, removes the `kz-var` prefix as it is not used anymore, renames `-rgb-params` to `-rgb`, and renames `kz-var-id-*` to `*-id`. E.g:
```

Comment:
I think let's keep this sentence simpler and rely on the examples. We might add new use cases to this rule in future.

```suggestion
- `kaizen/rename-tokens` - Reports and automatically renames any deprecated color names to the newer name. E.g:
```